conformance/metadata-fhirr4-service/README.md

```diff
@@ -1 +1,61 @@
 # FHIR R4 Metadata Service
+
+## Introduction
+This service provides implementation of FHIR Metadata API. This implements 
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+```Supported FHIR version is 4.0.1.```
+
```

Comment:
We missed to add how to build and run the service as a bal user in standlone mode?

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/8b982af37b97ff9cd7f6f9e8a273a8c6a9400010